docs/get_started/install.md

```diff
@@ -0,0 +1,347 @@
+```bash
+$ sudo apt-get install -y libopenblas-dev
+```
+
+**Step 3** Install OpenCV.
+
+MXNet uses [OpenCV](http://opencv.org/) for efficient image loading and augmentation operations.
+```bash
+$ sudo apt-get install -y libopencv-dev
+```
+
+**Step 4** Download MXNet sources and build MXNet core shared library.
+
+```bash
+$ git clone --recursive https://github.com/dmlc/mxnet
+$ cd mxnet
+$ make -j USE_OPENCV=1 USE_BLAS=openblas
+```
+
+*Note* - USE_OPENCV and USE_BLAS are make file flags to set compilation options to use OpenCV and BLAS library. You can explore and use more compilation options in `make/config.mk`.
+
+<br/>
+
+**Build the MXNet Python binding**
+
+**Step 1** Install prerequisites - python setup tools and numpy.
+
+```bash
+$ sudo apt-get install -y python-dev python-setuptools python-numpy
+```
+
+**Step 2** Build the MXNet Python binding.
+
+```bash
+$ cd python
+$ sudo python setup.py install
+```
+
+**Step 3** Validate the installation by running a simple MXNet code described [here](#validate-mxnet-installation).
+
+</div>
+
+</div>
+</div>
+</div>
+
+<!-- END - Linux Python CPU Installation Instructions -->
+
+# Validate MXNet Installation
+
+<div class="linux">
+<div class="python">
+<div class="cpu">
+
+<div class="pip build-from-source">
+
+Start the python terminal.
+
+```bash
+$ python
+```
+</div>
+
+<div class="docker">
+
+Launch a Docker container with `mxnet/python` image and run example MXNet python program on the terminal.
+
+```bash
+$ docker run -it mxnet/python bash # Use sudo if you skip Step 2 in the installation instruction
+
+# Start a python terminal
+root@4919c4f58cac:/# python
+```
+</div>
+
+<div class="virtualenv">
+
+Activate the virtualenv environment created for MXNet.
+
+```bash
+$ source ~/mxnet/bin/activate
+```
+
+After activating the environment, you should be seeing the prompt as below.
+
+```bash
+(mxnet)$
+```
+
+Start the python terminal.
+
+```bash
+$ python
+```
+
+</div>
+
+Run a short mxnet python program to create a 2X3 identity matrix, multiply each element in the matrix by 2 followed by adding 1. We expect output to be a 2X3 matrix with all elements being 3.
+
+```python
+>>> import mxnet as mx
+>>> a = mx.nd.ones((2, 3))
+>>> b = a * 2 + 1
+>>> b.asnumpy()
+array([[ 3.,  3.,  3.],
+       [ 3.,  3.,  3.]], dtype=float32)
+```
+
+</div>
+</div>
+</div>
+
+<!-- Clean up -->
+<div class="linux">
+<div class="python">
+<div class="cpu">
+
+<div class="pip build-from-source">
+
+Exit the Python terminal.
+
+```python
+>>> exit()
+$
+```
+</div>
+
+<div class="virtualenv">
+
+Exit the Python terminal and Deactivate the virtualenv MXNet environment.
```

Comment:
Is there a reason to capitalize Deactivate here? I think it should be "deactivate"